mysite/templates/Includes/LoginBar.ss

```diff
@@ -7,28 +7,28 @@
                 <div class="six columns">
                     <section class="login-form">
                         <p>Press login to enter CMS</p>
-                        <form action="/Security/LoginForm" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
+                        <form action="/Security/LoginForm?BackURL=%2Fadmin%2Fpages" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
```

Comment:
Is this easier than using a SS `Form` builder? Given it's open-source, kinda suggests bad practices, right? I guess the purpose of this code isn't for learning directly...